### PR TITLE
Tae8838 GitHub profile

### PR DESCRIPTION
**Pull request by tae8838**
## Issue

https://github.com/subeshb1/developer-community-stats/issues/260

## Description

Tae8838 GitHub profile

## Checklist

- [x] Added contributor object in ascending order in `contributors.json`
- [x] Github action build passing
- [x] No merge conflicts

### CODE REVIEW

File: readme-profiles/vunderkind.md [hunk 1]
```diff
@@ -18,6 +18,6 @@ What I'm currently interested in:
 - 3D
 - Interaction design
 
-<img src="https://github-profile-trophy.vercel.app/?username=vunderkind&theme=dracula&column=3&margin-w=15&margin-h=15 (https://github.com/ryo-ma/github-profile-trophy)">
+<img src="https://github-profile-trophy.vercel.app/?username=tae8838&theme=dracula&column=3&margin-w=15&margin-h=15 (https://github.com/ryo-ma/github-profile-trophy)">
```
Comment (subeshb1) [inline readme-profiles/vunderkind.md:21]: Lets revert the change in this fole

Comment (tae8838) [reply readme-profiles/vunderkind.md]: reverted. Thanks for the fast review!

File: readme-profiles/tae8838.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<h1 align="center">Hey, I'm Tae</h1>
+<h3 align="center">A Software Developer From Thailand</h3>
+
+<br/>
+
+- 🌱 I’m currently work mostly with **Ruby on Rails and Vue.js**
+
+
+
+<img src="https://github-profile-trophy.vercel.app/?username=subeshb1&theme=dracula&column=3&margin-w=15&margin-h=15 (https://github.com/ryo-ma/github-profile-trophy)">
```
Comment (subeshb1) [inline readme-profiles/tae8838.md:10]: You should change this to your username

Comment (tae8838) [reply readme-profiles/tae8838.md]: thanks for catching that. done

**Review by subeshb1 — APPROVED** [review]
